demo/src/lib/layout/playground/PlaygroundValue.svelte

```diff
@@ -26,38 +30,47 @@
 			/>
 		</div>
 	</td>
-	<td class="checkbox"
-		><button class="btn btn-link icon" title="Clear value" disabled={api.isEmpty} on:click={api.clear}
+	<td class="checkbox align-middle"
+		><button class="btn btn-link m-0 p-0 d-flex mx-auto" title="Clear value" disabled={api.isEmpty} on:click={api.clear}
 			><Svg className="icon-20" svg={clearIcon} /></button
 		></td
 	>
```

Comment:
Shouldn't this `<td>` with the `clearIcon` factorized instead of being repeated for each type of field?

Reply:
It is from old PR I could take care of it in the same PR I guess

Reply:
Indeed, it was already repeated before your PR, but as you did the same change several times (for each copy), maybe you can improve the code at the same time.